src/commcare_cloud/terraform/modules/commcarehq/main.tf

```diff
@@ -101,35 +103,50 @@ module "Redis" {
 #  # dns_domain       = "${var.dns_domain}"
 #}
 
-# If an RDS instance or cluster is needed, uncomment this.
-#module "database" {
-#  source                         = "../rds"
-#  vpc_id                         = "${module.network.vpc-id}"
-#  environment                    = "${var.environment}"
-#  dns_zone_id                    = "${var.dns_zone_id}"
-#  dns_domain                     = "${var.dns_domain}"
-#  azs                            = "${var.azs}"
-#  rds_storage                    = "${var.rds_storage}"
-#  rds_engine                     = "${var.rds_engine}"
-#  rds_engine_version             = "${var.rds_engine_version}"
-#  rds_instance_type              = "${var.rds_instance_type}"
-#  rds_database_name              = "${var.rds_database_name}"
-#  rds_username                   = "${var.rds_username}"
-#  rds_storage_type               = "${var.rds_storage_type}"
-#  rds_backup_window              = "${var.rds_backup_window}"
-#  rds_backup_retention           = "${var.rds_backup_retention}"
-#  rds_maintenance_window         = "${var.rds_maintenance_window}"
-#  rds_auto_minor_version_upgrade = "${var.rds_auto_minor_version_upgrade}"
-#  rds_multi_az                   = "${var.rds_multi_az}"
-#  rds_storage_encrypted          = "${var.rds_storage_encrypted}"
-#  rds_port                       = "${var.rds_port}"
-#  environment                    = "${var.environment}"
-#  rds_password                   = "${var.rds_password}"
-#  rds_prevent_destroy            = "${var.rds_prevent_destroy}"
-#  rds_instance_count             = "${var.rds_instance_count}"
-#  rds_sg_ip_ingress              = ["${module.generic-sg.security_group}","${module.network.g2-access-sg}","${module.openvpn.openvpn-access-sg}"]
-#  rds_subnet_a                   = "${module.network.subnet-a-db-private}"
-#  rds_subnet_b                   = "${module.network.subnet-b-db-private}"
-#  rds_subnet_c                   = "${module.network.subnet-c-db-private}"
-#  rds_instance_count             = "${var.rds_instance_count}"
-#}
+module "postgresql" {
+  source = "terraform-aws-modules/rds/aws"
+
+  identifier = "${local.rds["identifier"]}"
+
+  engine            = "postgres"
+  engine_version    = "9.6.6"
+  instance_class    = "${local.rds["instance_type"]}"
+  allocated_storage = "${local.rds["storage"]}"
+
+  name     = ""
+  username = "${local.rds["username"]}"
+  password = "${local.rds["password"]}"
+  port     = "${local.rds["port"]}"
+
+  iam_database_authentication_enabled = false
+
+  vpc_security_group_ids = ["${module.network.rds-sg}"]
+
+  maintenance_window = "${local.rds["maintenance_window"]}"
+  backup_window      = "${local.rds["backup_window"]}"
+  backup_retention_period = "${local.rds["backup_retention"]}"
+
+  # DB subnet group
+  subnet_ids = ["${module.network.subnet-a-db-private}",
+                "${module.network.subnet-b-db-private}",
+                "${module.network.subnet-c-db-private}"]
```

Comment:
are these supposed to appear in the list of subnets in `subnet_options`?

Reply:
No this is a different set of subnets altogether. Not totally sure why G2 set the DB subnets to be distinct from the app subnets, but again, I don't think it hurts. I don't have a strong enough sense of AWS networking (or networking best practices in general) to say that it's pointless, so for now I'm just leaving it as they set it up.